orion_mcp.py

```
@@ -108,26 +110,61 @@ def get_orion_configs() -> list[str]:
+        version: OpenShift version used to render the config template.
+
+    Returns:
+        A dictionary with "metrics" (list) and "meta" (per-metric metadata).
+    """
+    effective_config = config_name or config or "small-scale-udn-l3.yaml"
+    try:
+        metrics, meta_map = _load_config_metrics_with_meta(
+            os.path.join(ORION_CONFIGS_PATH, effective_config),
+            version=version,
+        )
+        return {"metrics": metrics, "meta": meta_map}
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)